tgui/packages/tgui/interfaces/Changelog.js

```diff
@@ -184,7 +184,7 @@ export class Changelog extends Component {
         </p>
         <p>
           {'Current project maintainers can be found '}
-          <a href="https://github.com/tgstation?tab=members">here</a>
+          <a href="https://github.com/orgs/tgstation/people">here</a>
```

Comment:
this links to a list of 90 people, and only a third (tops) are actual maintainers. should probably change the name to "A listing of those in our organization"...